app/src/main/java/dev/beefers/vendetta/manager/installer/step/download/DownloadVendettaStep.kt

```diff
@@ -17,7 +17,7 @@ class DownloadVendettaStep(
 
     override val nameRes = R.string.step_dl_vd
 
-    override val url: String = "https://github.com/vendetta-mod/VendettaXposed/releases/latest/download/app-release.apk"
+    override val url: String = "https://github.com/${preferenceManager.vendettaModuleRepo}/releases/latest/download/app-release.apk"
```

Comment:
While the repo is configurable, the asset name is hardcoded, which means other module repo releases have to name their releases the same way.